pcsx2/GS/GSState.cpp

```diff
@@ -3087,8 +3092,57 @@ __forceinline void GSState::HandleAutoFlush()
 			tex_coord.y = (int)((1 << m_context->TEX0.TH) * t);
 		}
 
-		const GSVector4i last_tex_page = tex_coord.xyxy() & page_mask;
-		const GSVector4i tex_page = tex_rect.xyxy() & page_mask;
+		const int clamp_minu = m_context->CLAMP.MINU;
+		const int clamp_maxu = m_context->CLAMP.MAXU;
+		const int clamp_minv = m_context->CLAMP.MINV;
+		const int clamp_maxv = m_context->CLAMP.MAXV;
+
+		if (m_context->CLAMP.WMS == 2)
```

Comment:
Should probably be a switch, or else if on WMS/WMT to avoid extra comparisons/branches when it's 0-1.